### ISSUE TITLE
Mojave: Crash when downloading any file

### ISSUE THREAD

**Issue by da2x** [brave/browser-laptop#14346]
1. Go to https://developer.apple.com/safari/download/(or any other download page)
2. Click to download the file, choose where to save it, and confirm

Expected: file download.

Actual: instant crash.

Brave: 0.22.727

Environment: macOS 10.14 Mojave beta (18A293u)

Stack trace: https://gist.github.com/da2x/401faa345503837684c62e304ddc9e46

**Comment by coasttech** [comment]
same issue

**Comment by coasttech** [comment]
No issue with Chrome

**Comment by da2x** [comment]
No issue with Vivaldi either (another Chromium browser).

**Comment by rojour** [comment]
same issue here.
No issues with Chrome, Chrome Canary, Opera Developer, Safari.

**Comment by LaurenWags** [comment]
+1 from community: https://community.brave.com/t/brave-crashes-to-desktop-on-download-macos/27280

**Comment by Hyracan** [comment]
Same here, issue only with Brave

**Comment by mdaneri** [comment]
Same problem here

**Comment by mitchwagner** [comment]
I also have this issue. MacOS 10.14 Beta. Brave 0.23.39. Firefox and Chrome do not have the problem.

**Comment by bsclifton** [comment]
Possibly fixed by https://github.com/brave/muon/commit/4504af1ad2a704729365f0bbab821c00ea357008 (which is scheduled for 0.23.x Release 5).  I'll assign that milestone so we can test

**Comment by LaurenWags** [comment]
@da2x @coasttech @rojour @Hyracan @mdaneri @mitchwagner mind downloading https://github.com/brave/browser-laptop/releases/tag/v0.23.74dev and seeing if this issue is resolved? note - there are some crashes in this build which we're working through viahttps://github.com/brave/browser-laptop/issues/14893 , so be advised if you give it a spin.

**Comment by LaurenWags** [comment]
from @rojour over on another issue, using 0.23.77 on Mojave is still crashing once the download is complete https://github.com/brave/browser-laptop/issues/14893#issuecomment-410307879

**Comment by eljuno** [comment]
+1 from community https://community.brave.com/t/mojave-beta-browser-force-closes-when-attempting-to-download-files/29133?u=eljuno

**Comment by bsclifton** [comment]
I don't have Mojave so I can't test this... but I suspect the issue is fixed with our new [developer channel release](https://brave.com/download-dev) (brave-core). Can someone please give this a shot?

Closing here as `wontfix` because we won't be fixing in this code-base

**Comment by mitchwagner** [comment]
@bsclifton

Using the following:
MacOS: 10.14 Beta (18A365a)
Brave: 0.23.105

I was able to download several files without issue.

I also tried the new developer channel release: 
MacOS: 10.14 Beta (18A365a)
Brave: Version 0.55.2 Chromium: 70.0.3528.4 (Official Build) dev (64-bit)

I was again able to download files without issue.

**Comment by bsclifton** [comment]
@mitchwagner awesome! Thanks for the update 😄 👍

**Comment by stshank** [comment]
No problems downloading on Brave Core Dev on MacOS Mojave beta 10 for me.

**Comment by kjozwiak** [comment]
awesome, thanks for checking/verifying @stshank 👍 Really appreciate the update.
